content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+## Overview
+
+{{< callout url="#" header="false" >}}
+Quality Gates Rules are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
```

Comment:
I added this just as an initial description, but I don't like it that much. @KickingTheTV do you have any suggestions on what to write for the Quality Gates overview?

Reply:
I think a definition of what a quality gate is and what the rules represent as the setup workflow would be good. Also we can maybe avoid using "workflow" specifically unless we are mentioning Workflow Automation (https://docs.datadoghq.com/service_management/workflows/) 🤔

Reply:
```suggestion
Quality Gates allows you to gate your workflows based on signals in Datadog.  You can create two types of rules types: Static Analysis and Tests.
```

Reply:
I changed it slighlty 😬